libs/cardano-ledger-conformance/test/Test/Cardano/Ledger/Conformance/Imp.hs

```diff
@@ -108,8 +118,30 @@ spec =
   withImpInit @(LedgerSpec Conway) $
     modifyImpInitProtVer @Conway (natVersion @10) $
       modifyImpInitExpectLedgerRuleConformance testImpConformance $ do
-        describe "Basic imp conformance" $
+        describe "Basic imp conformance" $ do
           it "Submit constitution" $ do
             _ <- submitConstitution @Conway SNothing
             passNEpochs 2
+          xit "P1 reference scripts must be witnessed" $ do
```

Comment:
Is the reason for this test to be added as disabled related to this issue in the spec https://github.com/IntersectMBO/formal-ledger-specifications/issues/626

Could you move this test into Babbage imp test spec instead, since it is more relevant there